packages/Symfony/tests/Rector/Console/ConsoleExecuteReturnIntRector/Fixture/return-external-function-call.php.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\Console\ConsoleExecuteReturnIntRector\Fixture;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ReturnExternalFunctionCall extends Command
+{
+    /** @var ExternalClass */
+    private $external;
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        return $this->external->doSomething();
+    }
+}
+
+final class ExternalCLass
+{
+    public function doSomething()
+    {
+        return null;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Rector\Console\ConsoleExecuteReturnIntRector\Fixture;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ReturnExternalFunctionCall extends Command
+{
+    /** @var ExternalClass */
+    private $external;
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        return $this->external->doSomething();
```

Comment:
How does Command work now with `null` value?

Maybe better would be retype to integer, e.g. `(int) $this->external->doSomething`

Chanding value in other method might break code elsewhere, where `null` is expected

```php
// app/SomeController.php

class SomeController
{
    public function go()
    {
        // this would not pass after the change
        return $this->external->doSomething() === null;
    }
}
```

Reply:
Yep i didn't thought about just casting the expressions, that makes the solution alos much cleaner and no need to for the `beforeTraverse()` anymore